python/runNanoSeq.py

```diff
@@ -0,0 +1,938 @@
+    cmd += "bgzip -f -l 2 %s/dsa/%s.dsa.bed; sleep 3; bgzip -t %s/dsa/%s.dsa.bed.gz;"%(tmpDir,i,tmpDir,i)
+    cmd += "touch %s/dsa/%s.done"%(tmpDir,i)
+    commands[i] =  ( cmd , )
+  
+  if ( args.index == None or args.index == 1 ) :
+    with open("%s/dsa/nfiles"%(tmpDir), "w") as iofile :
+      iofile.write(str( njobs ))
+
+  #execute dsa commans
+  print("Starting dsa calculation\n")
+  if ( args.index == None ) :
+    #multithread
+    with Pool( args.threads ) as p :
+      p.starmap( runCommand, commands )
+  else :
+    #array execution
+    runCommand( commands[args.index - 1][0] )
+  print("Completed dsa calculation\n")
+
+#var section
+if (args.subcommand == 'var' ) :
+  if (not os.path.isfile(tmpDir+'/dsa/args.json') ):
+    sys.exit("\nMust run dsa submmand prior to var\n")
+  else :
+    with open(tmpDir+'/dsa/args.json') as iofile :
+      oargs = json.load( iofile )
+
+  if (not os.path.isfile(tmpDir+'/dsa/nfiles') ):
+    sys.exit(tmpDir+'/dsa/nfiles not found!\n')
+  else :
+    with open(tmpDir+'/dsa/nfiles') as iofile :
+      nfiles = int(iofile.readline())
+  njobs = nfiles
+
+  for i in range(nfiles ) :
+    if ( len(glob.glob(tmpDir+"/dsa/%s.done"%i)) != 1 ) :
+      sys.exit("\ndsa job %s did not complete correctly\n"%i)
+    if ( len(glob.glob(tmpDir+"/dsa/%s.dsa.bed.gz"%i)) != 1 ) :
+      sys.exit("\ndsa job %s did not complete correctly\n"%i)
+
+  #make sure that number of jobs matches what was specified in part
+  if ( args.max_index != None ) :
+    #array execution
+    if ( args.max_index < njobs ) :
+      sys.exit("\nLSF array size must match number of jobs specified for part (%s)\n"%njobs)
+    if ( args.index > njobs ) :
+      print("\nWarning specified LSF array size is larger than jobs specified for part (%s)\n"%njobs)
+      sys.exit(0)
+  else :
+    #multithread
+    if ( args.threads < njobs ) :
+      sys.exit("\nNumber of threads must match number of jobs specified for part (%s)\n"%njobs)
+    if ( args.threads > njobs ) :
+      print("\nWarning number of threads is larger than jobs specified for part (%s)\n"%njobs)
+
+  commands = [ (None ,) ] * njobs
+  for i in range(njobs) :
+    #check for restarts
+    if ( os.path.isfile("%s/var/%s.done"%(tmpDir,i)) and \
+        os.path.isfile("%s/var/%s.var"%(tmpDir,i)) and \
+        os.path.isfile("%s/var/%s.cov.bed.gz"%(tmpDir,i)) ):
+      continue
+
+    #construct variantcaller commands
+    cmd = "variantcaller -B %s -U %s -O %s -a %s -b %s -c %s -d %s -f %s -i %s -m %s -n %s -p %s -q %s -r %s -v %s -x %s -z %s ;" \
+        %("%s/dsa/%s.dsa.bed.gz"%(tmpDir,i),"%s/var/%s.cov.bed"%(tmpDir,i),"%s/var/%s.var"%(tmpDir,i),
+          args.a,args.b,args.c, args.d, args.f, args.i, args.m, args.n, args.p, args.q, args.r, args.v, args.x, args.z)
+    cmd += "touch %s/var/%s.done"%(tmpDir,i)
+    commands[i] = ( cmd , )
+
+  if ( args.index == None or args.index == 1 ) :
```

Comment:
The code is very bracket heavy for python, it's not part of the accepted style, not a major issue but try to work towards the PEP standards.

```python3
if args.index is None or args.index == 1 :
```